Common/Brokerages/KrakenBrokerageModel.cs

```diff
@@ -94,20 +115,6 @@ public override bool CanSubmitOrder(Security security, Order order, out Brokerag
             }
             return base.CanSubmitOrder(security, order, out message);
         }
-        
-        /// <summary>
-        /// Kraken does no support update of orders
-        /// </summary>
-        /// <param name="security">Security</param>
-        /// <param name="order">Order that should be updated</param>
-        /// <param name="request">Update request</param>
-        /// <param name="message">Outgoing message</param>
-        /// <returns>Always false as Kraken does no support update of orders</returns>
-        public override bool CanUpdateOrder(Security security, Order order, UpdateOrderRequest request, out BrokerageMessageEvent message)
```

Comment:
Let's revert this change since it's not required, this version even has the missing docs 👍